stdlib/public/TensorFlow/ShapedArray.swift

```diff
@@ -889,3 +910,24 @@ extension ShapedArraySlice : CustomReflectable {
     return Mirror(self, children: [], displayStyle: .struct)
   }
 }
+
+/// Codable conformance.
```

Comment:
Triple-slash doc comments on extensions are not picked up by any doc generator or Xcode, are they?

Reply:
I'm sure they're not picked up by Jazzy, not sure about Xcode.
It would be better to use `// MARK`, probably. There are a bunch of triple slash comments in these files that should be changed to `// MARK`.